src/components/Button/button.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import styled from "@emotion/styled";
+import { keyframes } from "@emotion/core";
+
+interface ButtonProps {
+  text: string
+  type?: 'button' | 'submit' | 'reset'
+  isSubmitting?: boolean
+  isDisabled?: boolean
+}
+
+const Button = ({ text, type, isSubmitting, isDisabled }: ButtonProps) => (
```

Comment:
Return type missing, declare propType in the generics
```suggestion
const Button: React.FC<ButtonProps> = ({ text, type, isSubmitting, isDisabled }) => (
```